ivy/functional/frontends/paddle/tensor/tensor.py

```diff
@@ -139,6 +139,10 @@ def exp(self, name=None):
         return ivy.exp(self._ivy_array)
 
     @with_unsupported_dtypes({"2.4.2 and below": ("float16", "bfloat16")}, "paddle")
+    def subtract(self, other, name=None):
+        other_ivy = _to_ivy_array(other)
+        return ivy.subtract(self._ivy_array, other_ivy)
+

```

Comment:
You should add a new line with the decorator instead of using the previous one.

Reply:
@khushi-411  I don't understand the what you want me to do. Could you please elaborate?

Reply:
Please carefully check and address these comments. Notice the 6th line and check your functions.

Reply:
This point still needs to be addressed